kuma/static/js/analytics.js

```diff
@@ -93,20 +93,21 @@
         trackClientErrors: function() {
             $(win).on('error', function(e) {
                 var originalEvent = e.originalEvent;
-                analytics.trackError(' JavaScript Error: ' + originalEvent.message + ' ; ' + originalEvent.filename + ':' + originalEvent.lineno);
+                analytics.trackError('JavaScript Error', win.location.href, originalEvent.message + ' ; ' + originalEvent.filename + ':' + originalEvent.lineno);
```

Comment:
Sorry I didn't notice this in the email proposal until I saw it running in my console ...

Storing the `win.location.href` is duplicitous - GA already stores the `Page URL` dimension for every event. So, we could put something like `originalEvent.mesage`, `originalEvent.filename`, or `originalEvent.lineno` as the action if we think we may want to report on the data by one of those dimensions in addition to category and page.